src/pages/workspaces/workspace/notebooks/NotebookLauncher.js

```diff
@@ -42,7 +42,7 @@ const NotebookLauncher = _.flow(
     showTabBar: false
   })
 )(
-  ({ queryParams, notebookName, workspace, workspace: { workspace: { namespace, name }, accessLevel, canCompute }, runtimes, persistentDisks, refreshRuntimes },
+  ({ queryParams, notebookName, workspace, workspace: { workspace: { namespace, name, bucketName }, accessLevel, canCompute }, runtimes, persistentDisks, refreshRuntimes },
```

Comment:
TOL: (Existing issue, so no need to fix here) Why are we passing both workspace, and then a subset of the workspace fields? Could we remove the subset bit?